src/main.py

```diff
@@ -101,6 +104,73 @@
 
 # Endpoints
 
+
+def enunciado_ejercicio_a_html(ejercicio: str) -> str:
+    # Dark
+    # letra #c9d1d9 fondo #0d1117
+
+    # Light
+    # letra #111111 fondo fdfdfd
+
+    background_color = "#0d1117"
+    font_color = "#c9d1d9"
+
+    url = f"https://raw.githubusercontent.com/algoritmos-iii/ejercicios-2021-2c/main/{ejercicio}/Consigna.md"
+    req = requests.get(url)
+    md = markdown.markdown(
+        text=req.text,
+        extensions=['codehilite', 'fenced_code'],
+        extension_configs={'codehilite': {
+            'pygments_style': 'paraiso-dark',
+            'noclasses': True,
+            'nobackground': True,
+            'wrapcode': True,
+            'prestyles': 'overflow: auto'
+        }}
+    )
+    # html = jinja2_env.get_template('email_enunciado.html').render({
+    #     "bg_color": background_color,
+    #     "font_color": font_color,
+    #     "markdown": md
+    # })
+
+    html = f"""<table border="0" cellpadding="0" cellspacing="0" width="100%" bgcolor="#0d1117" style="color: {font_color}; background-color: {background_color}; padding: 20px;">
+    <tr>
+        <td>
+            {md}
+        </td>
+    </tr></table>"""
+    return html
+
+
+@app.route("/test-mail/enunciado")
+@use_args({
+    "ejercicio": fields.Str(required=True),
+})
+def test_email_enunciado_route(args):
+    ejercicio: str = args["enunciado"]
+    html = enunciado_ejercicio_a_html(ejercicio)
+    return flask.Response(html)
+
+
+@app.route("/send-mail/enunciado")
```

Comment:
Creo que no estamos nombrando bien los endpoints, ya que en la buena practica lo que se pone es `nombreDominio/` o `Entidad/` en plural. Luego la accion esta dicha con el verbo HTTP.

https://jorgegarcia.design/2020/11/08/good_practices_names/

Si observamos cuando estamos escribiendo un mail en gmail: `https://mail.google.com/mail/u/0/?fs=1&to=fiuba-algoritmos-iii-doc@googlegroups.com&tf=cm`. --> `/mail` y luego por query string el resto de los datos. 

Creo que podemos seguir esto y nombrar nuestros endpoints de esta manera --> `/mail/statement` , `/mail/exercise-grades` o `/mail/grades` y en vez de `/consultar` --> `/grades`

Reply:
Me gusta esa idea. Ahi hago los cambios.

Reply:
Lo unico que si, `/mail/exercise-feedback` no te gusta mas que `/mail/exercise-grades`?